### PR TITLE
Update k8s gcp

### PR DESCRIPTION
**Pull request by j-rewerts**

### CODE REVIEW

File: k8s/Production-GCP/initiatives-webapi-service.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+metadata:
+  name: initiatives-webapi
+  namespace: production
+  labels:
+    app: initiatives-webapi
+    technology: dotnetcore
+    tier: frontend
+    product: octava
+spec:
+  type: NodePort
```
Comment (dchenier) [inline k8s/Production-GCP/initiatives-webapi-service.yaml]: I think this should be of type "ClusterIP"...

File: k8s/UAT-GCP/wordpress-db-service.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (dchenier) [inline k8s/UAT-GCP/wordpress-db-service.yaml]: no longer need this file

File: k8s/Production-GCP/initiatives-webapi-service.yaml (hunk 1 above)
Comment (j-rewerts) [reply k8s/Production-GCP/initiatives-webapi-service.yaml]: GKE requires NodePort.

File: k8s/UAT-GCP/wordpress-db-service.yaml (hunk 2 above)
Comment (j-rewerts) [reply k8s/UAT-GCP/wordpress-db-service.yaml]: Resolved.

**Pull request merged**